index.js

```diff
@@ -91,6 +92,21 @@ module.exports = function(input, inputMap) {
 			});
 			return;
 		}
+
+		if (params.includeModulePaths) {
+			// Ensure module paths include full path from current working directory
+			var pwd = process.cwd();
+			if (context.substring(0, pwd.length) === pwd) {
```

Comment:
`if (context.substring(0, pwd.length) === pwd) {` felt like a helper based on the code below. Probably better wait on someone else's comment on this.